### DISCUSSION TITLE
ADS Requirements

### DISCUSSION THREAD

**Discussion by GonzaBasket98** [ODU-CGA-CubeSat/sealion-mission-architecture#34] [Ideas]
This discussion is for capturing requirements for the Attitude and Orbit Determination Systems.

**Comment by GonzaBasket98** [comment]
Orbital information from the spacecraft should provide insight as to whether is following the expected orbital profile or not.

**Reply by GonzaBasket98** [reply]
Orbital location (GPS and magnetometer) well-likely be known after first telemetry reading from spacecraft.

**Reply by capsulecorplab** [reply]
Knowledge of actual orbital profile (location & velocity vector) from spacecraft telemetry is necessary for predicting ground station overpass, in satisfying objective, _1.1 Establish UHF communication link with Virginia ground station_

**Comment by AstroNiashree** [comment]
#34 Meeting Updates
Design an ADC Model
(adapted the sensor models selected)
Simulation

**Comment by AstroNiashree** [comment]
- [ ] Follow the module on GPS and work with ADC

- [ ] Algorithm to reduce the dependency on GPS for power efficiency

**Reply by AstroNiashree** [reply]
F&G Functions
F&G series
Orbit Propagation
